packages/executable/package.json

```diff
@@ -13,15 +13,19 @@
   "pkg": {
     "assets": [
       "../client/build/**/*",
-      "../server/default_img/**/*"
+      "../server/**/*"
     ],
     "scripts": [
       "../server/addons/**/*"
-    ]
+    ],
+    "targets": [
+      "node14-macos-x64"
```

Comment:
This is fine for testing @lebaphi , but MS-Windows is the real desired target. I _believe_ that `pkg` can build MS-Windows executables under Macos.

Reply:
yep @IanMayo , it should be "node14", I will commit the change now